easybuild/tools/toolchain/compiler.py

```diff
@@ -27,6 +27,7 @@
 
 :author: Stijn De Weirdt (Ghent University)
 :author: Kenneth Hoste (Ghent University)
+:author: Damian Alvarez (Forschungszentrum Juelich GmbH)
```

Comment:
What is the policy here? Anybody that touches a file should be listed as an author, or is there a more specific criteria?